core/src/main/java/org/apache/calcite/tools/Frameworks.java

```diff
@@ -38,6 +37,7 @@
 import org.apache.calcite.sql2rel.SqlRexConvertletTable;
 import org.apache.calcite.sql2rel.SqlToRelConverter;
 import org.apache.calcite.sql2rel.StandardConvertletTable;
+import org.apache.calcite.statistic.MapSqlStatisticProvider;
 import org.apache.calcite.util.Util;
```

Comment:
Why we use a test MapSqlStatisticProvider for Frameworks

Reply:
That was a mistake. It was useful in tests, so we didn't notice that we'd made it the default. I have now made a query provider wrapped in a caching provider the default.